src/nomad_material_processing/general.py

```diff
@@ -449,36 +449,71 @@ class Miscut(ArchiveSection):
     The miscut in a crystalline substrate refers to
     the intentional deviation from a specific crystallographic orientation,
     commonly expressed as the angular displacement of a crystal plane.
+
+    It can be either directed along one of the crystallographic axes or in a specific
+    direction in the crystal defined by a main direction and the perpendicular one.
+
+    Refs.
+    https://dictionary.iucr.org/Miller_indices
+    https://dictionary.iucr.org/Direction_indices
+
+
     """
 
     angle = Quantity(
         type=float,
         description="""
-        The angular displacement from the crystallographic orientation of the substrate.
+        The miscut angle (or offcut angle, or angular displacement offset) toward
+        the crystallographic orientation of the substrate.
         """,
         a_eln=ELNAnnotation(
             component=ELNComponentEnum.NumberEditQuantity,
             defaultDisplayUnit='deg',
-            label='Miscut Angle',
         ),
         unit='deg',
     )
     angle_deviation = Quantity(
         type=float,
-        description='The ± deviation in the angular displacement.',
+        description='The ± deviation of the angular displacement offset.',
         a_eln=ELNAnnotation(
             component=ELNComponentEnum.NumberEditQuantity,
             defaultDisplayUnit='deg',
-            label='± Miscut Angle Deviation',
+            label='± Angle Deviation',
         ),
         unit='deg',
     )
-    orientation = Quantity(
+    hkl_reciprocal = Quantity(
+        type=str,
+        description="""Orientation of the miscut (or offcut).
+        The reciprocal lattice vector associated with the family of lattice planes is
+        OH = h a* + k b* + l c*, where a*, b*, c* are the reciprocal lattice basis
+        vectors. OH is perpendicular to the family of lattice planes and OH = 1/d where
+        d is the lattice spacing of the family.""",
```

Comment:
Indeed, this is the correct phrasing now.
Maybe just say "The OH vecotr is perpendicular to the family [...]"